com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -398,6 +468,10 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
         public override void SetupLights(ScriptableRenderContext context, ref RenderingData renderingData)
         {
             m_ForwardLights.Setup(context, ref renderingData);
```

Comment:
Is ForwardLights setting up some data for deferred as well?

Reply:
Yes, for forward-only pass.